sdk/spring/azure-spring-boot-samples/azure-spring-boot-sample-keyvault-certificates-server-side/script/setup.sh

```diff
@@ -0,0 +1,40 @@
+#!/usr/bin/env bash
+
+source script/export_environment_variables.sh
+
+# ==== set Azure Subscription ====
+az account set --subscription ${SUBSCRIPTION_ID}
+
+# ==== Create Resource Group ====
+az group create --name ${RESOURCE_GROUP_NAME} --location ${REGION_NAME}
+
+# ==== Create Key Vault, self-signed certificate ====
+az keyvault create --name ${KEY_VAULT_NAME}           \
+                   --resource-group ${RESOURCE_GROUP_NAME} \
+                   --location ${REGION_NAME}
+KEY_VAULT_URI=$(az keyvault show --name ${KEY_VAULT_NAME} --resource-group ${RESOURCE_GROUP_NAME} | jq -r '.properties.vaultUri')
+sed -i 's#export KEY_VAULT_URI=#&'"$KEY_VAULT_URI"'#' script/export_environment_variables_of_created_resource.sh
+
+az keyvault certificate create --vault-name ${KEY_VAULT_NAME} \
+    -n self-signed \
+    -p "$(az keyvault certificate get-default-policy)"
+
+# ==== Create Service Principal ====
+if [ ${SERVICE_PRINCIPAL_NAME} ];then
```

Comment:
Can we delete null check? Or we should check all the variables at the beginning of this script.
I prefer to delete the null check.

Reply:
I think we need not to check other vars for cli will throw exceptions when they are null. While for service principal, there is a case that when users want to use managed identity, so if we don't check for it then cli will report errors, which I think will make users confused?